packages/blocks/src/api/raw-handling/test/paste-handler.js

```diff
@@ -156,6 +176,93 @@ describe( 'pasteHandler', () => {
 		expect( result.isValid ).toBeTruthy();
 	} );
 
+	it( 'can preserve column alignments when pasting Markdown tables', () => {
```

Comment:
```suggestion
	it( 'can handle a table with cell alignments', () => {
```

Nit: This test is not limited to markdown data only.